src/style.css

```diff
@@ -0,0 +1,46 @@
+:root {
+  --star-size: 16px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  box-sizing: border-box;
```

Comment:
A better practice is to place this property in a * selector and remove it from all other places.

Reply:
are talking about margin? Or box-sizing?